### PR TITLE
[REF][8.0][hr_payroll_account] Get partner_id with a function to inheritable case

### PR DESCRIPTION
**Pull request by moylop260**
Proposal to add a REF - More info here: https://github.com/odoo/odoo/issues/3274
OPW ticket 616565

### CODE REVIEW

**Comment by jorgenaranjo** [comment]
Hello @mart-e 

I did make the requested changes, you can check the PR again.

Thank you!

File: addons/hr_payroll_account/hr_payroll_account.py [hunk 1]
```diff
@@ -26,6 +26,38 @@
...
+    '''
+    _inherit = 'hr.payslip.line'
+
+    def _get_partner_id(self, cr, uid, ids, credit_account, context=None):
+        """
+        Get partner_id of slip line to use in account_move_line
+            and adds the ability to inherit it
+        """
+        res = {}
+        payslip_line = self.browse(cr, uid, ids[0], context=context)
```
Comment (mart-e) [inline addons/hr_payroll_account/hr_payroll_account.py]: if you are supporting only one id, don't make your argument a list

**Comment by mart-e** [comment]
Thanks for the update.
Could you also update Vauxoo's CLA ?

**Comment by moylop260** [comment]
Hello @mart-e 
cla update: https://github.com/odoo/odoo/pull/7039

**Comment by mart-e** [comment]
Thanks, merged the CLA update.
Made a few changes (recordset as parameter) and applied on master at 075d80a. Will be merged in master soon.

**Comment by moylop260** [comment]
@mart-e,
Thank you

**Comment by jorgenaranjo** [comment]
@mart-e 
The change for receive the id in _get_partner_id function was made.

**Comment by mart-e** [comment]
Actually the patch was arleady ready at 075d80afb94a2819e29e1cbb1c815593e65b4268 but I was just waiting for the master branch to be back online.
Now it's done so I have merged it at 6b5609174957b7dd1b7d1701a16b00e239c7e17c
Thanks

**Pull request closed without merging**